### ISSUE TITLE
Automatically recalculate the value when the formula is changed

### ISSUE THREAD

**Issue by damagedgoods** [chazzdesign/sketch-calculator#16]

**Comment by damagedgoods** [comment]
I'm having some problems with the Action API. I think there might be a bug, as the rename event is not always fired.

**Comment by damagedgoods** [comment]
Ale confirmed that renaming action is not working properly. I think it might be confusing autocalculating some actions (value edition) and not others (renaming, etc). So maybe for the moment we should remove autocalculate from every place, and always launch the calculation on demand. At least until they solve the issue.

What do you think, @javierarce ?

**Comment by javierarce** [comment]
Yes, I agree that removing it momentarily it's the best solution for now 👌
